valhalla/app/javascript/mixins/manifesto-filemanager-mixins.js

```diff
@@ -0,0 +1,58 @@
+const ManifestoFilemanagerMixins = {
+
+  mainSequence: function () {
+    // mainSequence is the one without an id (not ideal since it could have an id)
+    const s = this.getSequences()
+    const main_sequence = s.filter((seq) => seq.id !== 'undefined')
+    return main_sequence[0]
+  },
+
+  getCanvasMainThumb: function (canvas) {
+    const images = canvas.getImages()
+    var thumb = "http://via.placeholder.com/300x400"
```

Comment:
Should this be a protocol-relative URL specified here? (Then again, please see https://jeremywagner.me/blog/stop-using-the-protocol-relative-url/).  Also, could this be webpacked or compiled as a static asset (as in http://clarkdave.net/2015/01/how-to-use-webpack-with-rails/#including-precompiled-assets-in-views)?

Reply:
Yes, created a separate issue for this: https://github.com/pulibrary/figgy/issues/613